spec/system/user_card_badges_spec.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+RSpec.describe "User card badges plugin", type: :system do
+  before { SiteSetting.user_card_badges_enabled = true }
+  let(:user) { Fabricate(:user) }
+  let!(:topic) { Fabricate(:topic, user: user) }
+  let(:badge) { Fabricate(:badge, image_upload: Fabricate(:upload)) }
```

Comment:
shall we make it more`fab`-ulous? 😛 

```suggestion
  fab!(:user)
  fab!(:topic) { Fabricate(:topic, user: user) }
  fab!(:badge) { Fabricate(:badge, image_upload: Fabricate(:upload)) }
```

Reply:
I mean... it doesn't actually make a difference since there's only one test. In fact, it's probably slower to use `fab!` in this case since it has to set up an unnecessary PG savepoint?

But I will accept for consistency's sake 😛